sample/github/README.md

```diff
@@ -62,89 +95,85 @@ stringData:
 
+You can inspect those resources with the following `kubectl` commands:
+
+```shell
 # This will show the available EventSources that you can generate feeds from:
 kubectl get eventsources -o yaml
 
 # This will show the available EventTypes that you can generate feeds from:
 kubectl get eventtypes -o yaml
-
 ```
 
-To make this service accessible to github, we first need to determine its ingress address
-(might have to wait a little while until 'ADDRESS' gets assigned):
+Configure the `Flow` to point to a GitHub repository you are able to control and update
+`spec.trigger.resource` in `samples/github/flow.yaml`.  
+
+And then apply the resources:
+
 ```shell
-$ kubectl get ingress --watch
-NAME                      HOSTS                           ADDRESS           PORTS     AGE
-git-webhook-ingress   git-webhook.default.aikas.org   104.197.125.124   80        12m
-```
+ko apply -f sample/github/auth.yaml
+ko apply -f sample/github/flow.yaml
+```   
 
-Once the `ADDRESS` gets assigned to the cluster, you need to assign a DNS name for that IP address. This DNS address needs to be:
-git-webhook.default.<domainsuffix you created> so for me, I would create a DNS entry from:
-git-webhook.default.aikas.org pointing to 104.197.125.124
-[Using GCP DNS](https://support.google.com/domains/answer/3290350)
+Once deployed, you can inspect the created resources with `kubectl` commands:
 
-So, you'd need to create an A record for git-webhook.default.aikas.org pointing to 104.197.125.124
+```shell
+# This will show the available Flows we created:
+kubectl get flows -o yaml
 
-To now send events via the github webhook for pull requests to the function we created above, you need to
- create a Feed object. Modify sample/github/feed.yaml to point to owner of the repo as well
- as the particular repo you want to subscribe to. So, change spec.trigger.resource with the owner/repo
- you want.
+# This will show the available Feeds created from the Flow:
+kubectl get feeds -o yaml
 
- For example, if I wanted to receive notifications to:
- github.com/inlined/robots repo, my Feed object would look like so:
+# This will show the available Channels created from the Flow:
+kubectl get channels -o yaml
 
-```yaml
-apiVersion: knative.dev/v1alpha1
-kind: Feed
-metadata:
-  name: feed-example
-  namespace: default
-spec:
-  trigger:
-    service: github
-    eventType: pullrequest
-    resource: inlined/robots
-    parametersFrom:
-      - secretKeyRef:
-          name: githubsecret
-          key: githubCredentials
-  action:
-    routeName: git-webhook
+# This will show the available Subscriptions created from the Flow:
+kubectl get subscriptions -o yaml
+
+# This will show the available ClusterBuses we created in the prerequisites:
+kubectl get clusterbuses -o yaml
 ```
 
-Then create the secret and a feed so that you can see changes
+Then create the secret so that you can see changes:
 
 ```shell
  ko apply -f sample/github/githubsecret.yaml
```

Comment:
Does it actually work to create the secret after creating the flow?

Reply:
yeah, it should fail until the secret gets created.